src/main/java/Dross.java

```diff
@@ -0,0 +1,85 @@
+import java.util.Scanner;
+
+public class Dross {
+
+    //Size 100 array tasklist for Dross
+    private static List drossTaskList = new List();
+    //Method that prints line using "_" char
+    public static void printLine() {
+        for (int i = 0; i < 55; i++) {
+            System.out.print("_");
+        }
+        System.out.println();
+    }
+
+    //Method that echoes back whatever user typed in provide that it is not "bye"
+    public static void echo(){
+        Scanner in = new Scanner(System.in);
+        String line = in.nextLine();
+
+        while (!line.equals("bye")) {
+            printLine();
+            System.out.println(line);
+            printLine();
+            line = in.nextLine();
+        }
+        goodbye();
+    }
+
+    //Method that prints bye message
+    public static void goodbye(){
```

Comment:
The method name `goodbye` should be a verb. Since this method just consists of print statements, perhaps consider an alternative name like `printGoodbyeMessage`?